Add ability to customize the ready screen on empty admins (#5441)
marmelab/react-admin: docs/Admin.md

);
```

### `ready`

When you run an `<Admin>` with no child `<Resource>`, react-admin displays a "ready" screen:

![Empty Admin](./img/tutorial_empty.png)

> You can replace that "ready" screen your custom component by passing it as the `ready` prop:

Reviewer: ```suggestion
You can replace that "ready" screen by passing a custom component as the `ready` prop:
```
